### DISCUSSION TITLE
Add relational fields to Insights

### DISCUSSION THREAD

**Discussion by lucameusburger** [directus/directus#14218] [Feature Requests]
It would be great to select related fields in insights.

For example: 

I have a collection 'events' and 'days'. A day has a date can have multiple events. Events only have times.
Now i want to make a graph that shows how many events are on one day. Select the collection event but here i cant select event.day.date.

**Comment by lucameusburger** [comment]
Sorry if that disscussion is redundant . I already started one that points the same thing out in views. 
https://github.com/directus/directus/discussions/13153

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
